### DISCUSSION TITLE
semanticCommitType is not honored

### DISCUSSION THREAD

**Discussion by electriquo** [renovatebot/renovate#14702] [Request Help] [answered]
i am using non self-hosted renovatebot, and it works fire for me.
i wanted to tweak the commit type, so i have changed [`semanticCommits`](https://docs.renovatebot.com/configuration-options/#semanticcommits) to `enabled`, and [`semanticCommitType`](https://docs.renovatebot.com/configuration-options/#semanticcommittype) to `fix`.

but renovate keeps opening pull-request with `chore` commit type.

this is my configuration

```json
{
  "extends": [
    "config:base"
  ],
  "semanticCommits": "enabled",
  "semanticCommitType": "fix",
  "semanticCommitScope": "deps"
}
```

running renovatebot locally, it seems that the configuration is evaluated correctly.
```shell
INFO: Repository started (repository=foo/bar)
   "renovateVersion": "32.6.2"
...
DEBUG: Repository config (repository=foo/bar)
       "fileName": ".github/renovate.json",
       "config": {
         "extends": ["config:base"],
         "semanticCommits": "enabled",
         "semanticCommitType": "fix",
         "semanticCommitScope": "deps"
       }
...
INFO: Repository finished (repository=foo/bar)
      "durationMs": 17593
DEBUG: Renovate exiting
```

this issue persist in all of my repositories.
1. how can i check why `semanticCommitType` is not honored by renovatebot? is this a bug?
2. where is [`config.js`](https://github.com/renovatebot/renovate/blob/5200663cdd0f055637d46aa7604b3a976391dbf7/docs/usage/examples/self-hosting.md#L235-L244) option documentation? (same as [`renovate.json` configuration options](https://docs.renovatebot.com/configuration-options))

**Comment by rarkins** [comment, marked as answer]
config:base has an opinionated preset relating to semantic commits which is taking priority. If you look it up in the docs you can see which one it is, and you can use ignorePresets if you wish to have it ignored.

**Reply by electriquo** [reply]
> If you look it up in the docs you can see which one it is

could you please include a reference? 

> use ignorePresets if you wish to have it ignored

[reading through the docs](https://docs.renovatebot.com/configuration-options/#ignorepresets), will adding `ignorePresets: [":semanticPrefixFixDepsChoreOthers"]` to the configuration is enough for the `semanticCommitType: "fix"` to be active\take precedence? if not, a snippet will be appreciated.

when i run it locally after adding the above, where in the logs i can see how it is evaluated eventually?

**Reply by rarkins** [reply]
That should be all you need, you can compare the before/after by running it with `printConfig: true` in your config as it will dump your full configuration including resolved presets

**Reply by electriquo** [reply]
@rarkins: seems like it solved the issue.

# THANK YOU
i would like to **thank you** for all your amazing work and guidance all over.

to have a full closure, could you please relate to
> where is [config.js](https://github.com/renovatebot/renovate/blob/5200663cdd0f055637d46aa7604b3a976391dbf7/docs/usage/examples/self-hosting.md#L235-L244) option documented? (same as [renovate.json configuration options](https://docs.renovatebot.com/configuration-options))

**Reply by rarkins** [reply]
https://docs.renovatebot.com/getting-started/running/#global-config

**Reply by electriquo** [reply]
thanks. maybe i am missing something but the documentation does not say anything about the configuration options, such as `endpoint`, `platform`, `labels`, `logLevel`, `repository`, etc.

so i am wondering where can i read more about it?

**Reply by rarkins** [reply]
Type "endpoint" into the search bar and follow the first link
